### ISSUE TITLE
react 17.0

### ISSUE THREAD

**Issue by tapkain** [kotarella1110/use-simple-infinite-scroll#164]
Hello, I want to try the library but npm cannot resolve the dependency for react, can you update to the latest deps? Thanks!

**Comment by kotarella1110** [comment]
@tapkain Thanks for your feedback!
This library doesn't depend on other libraries.
Please see here: https://david-dm.org/kotarella1110/use-simple-infinite-scroll

Can you provide me with example (CodeSandbox or repository) to reproduce it?

**Comment by kotarella1110** [comment]
@tapkain Thanks for your feedback!
Are you referring to the warning output at the time of npm install?

<img width="686" src="https://user-images.githubusercontent.com/12913947/100365422-cec62900-3042-11eb-9817-5754886da2df.png">

**Comment by tapkain** [comment]
I got this when running npm install

```npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: project@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from use-simple-infinite-scroll@1.0.1
npm ERR! node_modules/use-simple-infinite-scroll
npm ERR!   use-simple-infinite-scroll@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.```

**Comment by kotarella1110** [comment]
@tapkain Thanks! I'll fix this soon 🙏

**Comment by kotarella1110** [comment]
@tapkain I have created a PR, can you review it?

**Comment by tapkain** [comment]
Thank you for quick reply, wish success with this project!

**Comment by kotarella1110** [comment]
@tapkain I have previously unpublished v1.0.2 and [cannot seem to publish it](https://github.com/kotarella1110/use-simple-infinite-scroll/runs/1459681097?check_suite_focus=true).
Please wait for v1.1.0 to be released 🙏 I'll do this soon.

**Comment by kotarella1110** [comment]
:tada: This Issue is included in version 1.1.0 :tada:

The release is available on:
- [GitHub release](https://github.com/kotarella1110/use-simple-infinite-scroll/releases/tag/v1.1.0)
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/use-simple-infinite-scroll/v/1.1.0)
